sharepa/analysis.py

```diff
@@ -16,6 +16,33 @@ def bucket_to_dataframe(name, bucket):
     ).T
 
 
+def convert_key(x):
+    x.key = x.key_as_string
+    return x
+
+
+def aggregation_to_dataframe(name,agg,filter=None):
```

Comment:
```filter``` is actually a python function, so you should call this variable something else. Also, would it be possible to take the Response.aggregations and find all name-bucket pairs?